### ISSUE TITLE
Downloading data sets and pre-trained models

### ISSUE THREAD

**Issue by notknowchild** [NVIDIA/vid2vid#151]
I downloaded the data set and the pre-trained model and the connection timed out. I found a link in the download_gdrive.py in the scripts but it didn't open. Is there any other way to download the data set and pre-trained model? Thank you.

**Comment by dkosovic** [comment]
The issue is with Google drive which comes up with a "Google Drive can't scan this file for viruses" as the zip files are too large.

The files can be downloaded with a web browser by clicking on the _"Download anyway"_ button on the following pages :

datasets.zip (202MB) :
* https://docs.google.com/uc?export=download&id=1rPcbnanuApZeo2uc7h55OneBkbcFCnnf

Cityscapes_models_all.zip (2.9GB) :
* https://docs.google.com/uc?export=download&id=1MKtImgtnGC28EPU7Nh9DfFpHW6okNVkl

face_models.zip (1.4GB)
* https://docs.google.com/uc?export=download&id=10LvNw-2lrh-6sPGkWbQDfHspkqz5AKxb

**Comment by sumit171993** [comment]
No, these links (1st for dataset.zip and the other one Cityscape link) not working. 
Both links are giving forbidden error: ![Image](https://github.com/user-attachments/assets/be4da09d-341f-4644-859f-a10d4106cbcf)


Please do the needful

**Comment by xiaomaofei** [comment]
same questions。help me

**Comment by sumit171993** [comment]
> same questions。help me

Also I am facing issue in installation for flownet_pytorch. @xiaomaofei  able to install it? 
Getting [resample2d_cuda](https://github.com/NVIDIA/vid2vid/tree/master/models/flownet2_pytorch/networks/resample2d_package) installation issue, because of compatibility between Cuda Version and Pytorch version. Cuda_Version is 12.6 but pytorch version is 11.8. 

Thanks and Regards

**Comment by dkosovic** [comment]
> * datasets.zip (202MB) : https://docs.google.com/uc?export=download&id=1rPcbnanuApZeo2uc7h55OneBkbcFCnnf
> * Cityscapes_models_all.zip (2.9GB) :  https://docs.google.com/uc?export=download&id=1MKtImgtnGC28EPU7Nh9DfFpHW6okNVkl
> * face_models.zip (1.4GB) : https://docs.google.com/uc?export=download&id=10LvNw-2lrh-6sPGkWbQDfHspkqz5AKxb

Although I previously posted the above links, unfortunately I no longer have local copies of the files to be able to upload elsewhere.

**Comment by sumit171993** [comment]
@dkosovic Thanks for the links. But it still does not work for me. 
Can you also help me with the flownet_pytorch installation issue which I am facing?

Regards,

**Comment by sumit171993** [comment]
@dkosovic This is the issue: https://github.com/NVIDIA/vid2vid/issues/151#issuecomment-2490208552

**Comment by dkosovic** [comment]
@sumit171993 I just requoted the links I previously posted in this issue. I didn't mean to imply they worked, sorry about that. Back in 2022 I posted them after someone in my workplace was having the issue downloading them with the Python script downloader. So, I worked out what the actual links were by looking at the code and shared the links as I thought they might be useful for others.

Unfortunately, I no longer have local copies of the zip files to share.

Sorry I can't help with the `flownet_pytorch` issue, it's been ages since I last installed it and can't remember much.

**Comment by sumit171993** [comment]
It's OK, thanks!!
